index.js

```diff
@@ -115,55 +110,55 @@ program
     // reformat if needful:
     userPrefs = {};
     Object.keys(_userPrefs).forEach(function (k) {
-      v = _userPrefs[k];
-      var prefix = "+";
+      var v = _userPrefs[k];
+      var prefix = '+';
       var n = prefix.length;
-      if (k.indexOf(prefix) == 0) { //
-        k = "extensions." + id + "." + k.slice(n)
+      if (k.indexOf(prefix) === 0) {
+        k = 'extensions.' + id + '.' + k.slice(n);
       }
       userPrefs[k] = v;
-    })
+    });
 
-    console.info("--prefs: \n%s", JSON.stringify(userPrefs,null, 2));
+    console.info('--prefs: \n%s', JSON.stringify(userPrefs,null, 2));
     // set special addon prefs
     ourPrefs = {};
     if (variation) {
-      ourPrefs[prefsBr + ".shield.variation"] = variation;
-      console.info("setting variation to: %s", f.bold(variation));
+      ourPrefs[prefsBr + '.shield.variation'] = variation;
+      console.info('setting variation to: %s', f.bold(variation));
     } else {
-      console.info("choosing variation %s", f.bold("at random"));
+      console.info('choosing variation %s', f.bold('at random'));
     }
 
     if (options.firstrun) {
-      ourPrefs[prefsBr + ".shield.firstrun"] = options.firstrun;
-      console.info("firstrun: mimic previous run: %s", f.bold(options.firstrun))
+      ourPrefs[prefsBr + '.shield.firstrun'] = options.firstrun;
+      console.info('firstrun: mimic previous run: %s', f.bold(options.firstrun));
     } else {
-      console.info("firstrun: simulate firstrun as %s", f.bold("NOW"))
+      console.info('firstrun: simulate firstrun as %s', f.bold('NOW'));
     }
 
     if (options.debug) {
       ourPrefs['shield.debug'] = true;
-      console.info("setting `shield.debug`");
+      console.info('setting `shield.debug`');
     }
 
     var newPrefs = merge(standardPrefs, ourPrefs, userPrefs);
 
-    var file = tempfile(".json")
-    jsonfile.writeFileSync(file, newPrefs, {spaces: 2})
-    console.info("combined prefs: %s\n%s",
+    var file = tempfile('.json');
+    jsonfile.writeFileSync(file, newPrefs, {spaces: 2});
+    console.info('combined prefs: %s\n%s',
         f.link(file),
         JSON.stringify(newPrefs,null,2));
 
     //console.info("jpm", "run", "code:", addonDir, "variation:", variation, "--prefs", options.prefs, passedOn);
-    var callArgs = ["jpm", "run", '--addon-dir', addonDir, "--prefs", file].concat(passedOn);
-    console.info("jpm args:\n%s", util.inspect(callArgs) );
+    var callArgs = ['jpm', 'run', '--addon-dir', addonDir, '--prefs', file].concat(passedOn);
+    console.info('jpm args:\n%s', util.inspect(callArgs) );
     nodeCLI.exec.apply(nodeCLI, callArgs);
-    exit(0);
-  })
+    process.exit(0);
```

Comment:
This was complaining that `exit()` was undefined. I took a guess and assumed you meant [`process.exit()`](https://nodejs.org/api/process.html#process_process_exit_code).